lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
I’ve removed the extra block scope here and `const`’ed the variables, that should be okay (but if anyone wants something else very strongly, I’ll change that again. I don’t really care that much either).